translator.go

```diff
@@ -17,36 +21,63 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type Catalog struct {
+	Repositories []string `json:"repositories"`
+}
+
 func checkImage(ctx context.Context, tagName string, cfg Config) (bool, error) {
 	if cfg.IsLocal {
-		apiClient, err := client.NewClientWithOpts(
-			client.WithAPIVersionNegotiation(),
-		)
-		defer apiClient.Close()
-		if err != nil {
-			return true, err
-		}
+		if cfg.Local.Driver == "minikube" {
+			resp, err := http.Get(fmt.Sprintf("http://localhost:%d/v2/_catalog", cfg.Local.RegistryPort))
```

Comment:
Everything else is related to communication inside kubernetes, as such we use the RegistryDomain. However, this is different here because we are trying to hit the docker registry which is port-forwarded. For orbstack, we call it directly rather than call the registry.